iina/MPVController.swift

```diff
@@ -664,6 +664,18 @@ class MPVController: NSObject {
     case MPV_EVENT_COMMAND_REPLY:
       let reply = event.pointee.reply_userdata
       if reply == MPVController.UserData.screenshot {
+        let code = event.pointee.error
+        guard 0 <= code else {
```

Comment:
I think I would prefer the condition to be flipped

```suggestion
        guard code >= 0 else {
```

Reply:
Fixed